### ISSUE TITLE
New - Danish language (complete based on ver. 2.4.7)

### ISSUE THREAD

**Issue by EV-soft** [prasathmani/tinyfilemanager#721]
```

{
  "appName": "Tiny File Manager",
  "version": "2.4.7",
  "language": [
    {"name": "Dansk",
      "code": "da",
      "translation": {
        "Access denied. IP restriction applicable": "Adgang nægtet. IP-begrænsning gælder",
        "Actions": "Handlinger",
        "Advanced Search": "Avanceret søgning",
        "AdvancedEditor": "Advanceret Editor",
        "AppTitle": "Filhåndtering",
        "Archive not created": "Arkiv er ikke oprettet",
        "Archive not unpacked": "Arkiv er ikke pakket ud",
        "Archive unpacked": "Arkiv er udpakket",
        "Archive": "Arkiv",
        "Back": "Tilbage",
        "BackUp": "Backup",
        "CalculateFolderSize": "Beregn mappestørrelse",
        "Cancel": "Afbryd",
        "Change": "Ændre",
        "ChangePermissions": "Ændre tilladelser",
        "Check Latest Version": "Tjek seneste version",
        "Copied from": "Kopieret fra",
        "Copy": "Kopi",
        "CopyTo": "Kopier til",
        "Copying": "Kopierer",
        "Create archive?": "Opret arkiv?",
        "CreateNewItem": "Opret ny",
        "CreateNow": "Opret nu",
        "Created": "Oprettet",
        "Delete selected files and folders?": "Slet valgte filer og mapper?",
        "Delete": "Slet",
        "Deleted": "Slettet",
        "DestinationFolder": "Destinationsmappe",
        "DirectLink": "Direkte link",
        "Download": "Hent",
        "Edit": "Rediger",
        "Error while copying from": "Fejl ved kopiering fra",
        "Error while deleting items": "Fejl ved sletning af elementer",
        "Error while fetching archive info": "Fejl ved hentning af arkivoplysninger",
        "Error while moving from": "Fejl ved flytning fra",
        "Error while renaming from": "Fejl ved omdøbning fra",
        "ErrorReporting": "Fejlrapportering",
        "Execute": "Udfør",
        "FILE EXTENSION HAS NOT SUPPORTED": "FILEXTENSION ER IKKE UNDERSTØTTET",
        "File Saved Successfully": "Fil blev gemt",
        "File extension is not allowed": "Filtypenavn er ikke tilladt",
        "File not found": "Fil ikke fundet",
        "File or folder with this path already exists": "Fil eller mappe med denne sti findes allerede",
        "File": "Fil",
        "Files": "Filer",
        "Folder is empty": "Mappen er tom",
        "Folder": "Folder",
        "FreeOf": "Ledig af",
        "FullSize": "Fuld størrelse",
        "Generate new password hash": "Generer ny adgangskode-hash",
        "Generate": "Generer",
        "Group": "Gruppe",
        "Help Documents": "Hjælpedokumenter",
        "Help": "Hjælp",
        "HideColumns": "Skjul kolonner",
        "Invalid characters in file name": "Ugyldige tegn i filnavn",
        "Invalid characters in file or folder name": "Ugyldige tegn i fil- eller mappenavn",
        "Invalid file or folder name": "Ugyldigt fil- eller mappenavn",
        "InvertSelection": "Inverter valget",
        "ItemName": "Emne navn",
        "ItemType": "Emne type",
        "Language": "Sprog",
        "Login failed. Invalid username or password": "Login mislykkedes. Ugyldigt brugernavn eller adgangskode",
        "Login": "Log på",
        "Logout": "Log ud",
        "Modified": "Ændret",
        "Move": "Flyt",
        "Moved from": "Flyttet fra",
        "Name": "Navn",
        "NewItem": "Nyt emne",
        "NormalEditor": "Standard Editor",
        "Nothing selected": "Intet valgt",
        "Open": "Åben",
        "Operations with archives are not available": "Operationer med arkiver er ikke tilgængelige",
        "Other": "Andet",
        "Owner": "Ejer",
        "PartitionSize": "Partitionsstørrelse",
        "Password": "Adgangskode",
        "Paths must be not equal": "Stierne må ikke være ens",
        "Permissions changed": "Tilladelser ændret",
        "Permissions not changed": "Tilladelser ikke ændret",
        "Perms": "Tilladelser",
        "Preview": "Forhåndsvisning",
        "Read": "Læs",
        "Rename": "Omdøb",
        "Renamed from": "Omdøbt fra",
        "Report Issue": "Rapportér problem",
        "Root path": "Rod mappe",
        "Save": "Gem",
        "Search file in folder and subfolders...": "Søg fil i mappe og undermapper...",
        "Search": "Søg",
        "Select folder": "Vælg mappe",
        "SelectAll": "Vælg alle",
        "Selected files and folder deleted": "Valgte filer og mappe slettet",
        "Settings": "Indstillinger",
        "ShowHiddenFiles": "Vis skjulte filer",
        "Size": "Størrelse",
        "Source path not defined": "Kildesti er ikke defineret",
        "SourceFolder": "Kildemappe",
        "Tar": "Tar",
        "Theme": "Tema",
        "UnSelectAll": "Fravælg alle",
        "UnZip": "Pak ud",
        "UnZipToFolder": "Pak ud i mappe",
        "Upload": "Upload",
        "UploadingFiles": "Uploader filer",
        "Username": "Brugernavn",
        "Write": "Skriv",
        "You are logged in": "Du er logget ind",
        "Zip": "Zip",
        "already exists": "eksisterer allerede",
        "dark": "mørkt",
        "light": "lyst",
        "not created": "ikke oprettet",
        "not deleted": "ikke slettet",
        "not found!": "ikke fundet!",
        "password_hash not supported, Upgrade PHP version": "password_hash er ikke understøttet, opgrader PHP-versionen",
        "to": "til"
      }                                                     
    }
  ]
}

```

**Comment by prasathmani** [comment]
@EV-soft , can you create a PR

**Comment by EV-soft** [comment]
No...

**Comment by AgentSmith0** [comment]
I created one: https://github.com/prasathmani/tinyfilemanager/pull/728

**Comment by prasathmani** [comment]
merged #728
